Sign out button in the account console has wrong Selenium locator

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

testsuite

### Describe the bug

After merging the https://github.com/keycloak/keycloak/pull/29270, it is not possible to locate the `Sign out` button in the account console.

The locator in the `LoggedInPageHeader` is no longer valid.

https://github.com/keycloak/keycloak/blob/main/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/webauthn/pages/fragments/LoggedInPageHeader.java#L30

```
WebAuthnErrorTest ++ 08:32:36,037 INFO  [org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest] [WebAuthnErrorTest] errorPageWithTimeout() STARTED
WebAuthnErrorTest ++ Starting ChromeDriver 125.0.6422.78 (14db42ec38aded3304a3e624a0a038e02956b87e-refs/branch-heads/6422@{#1088}) on port 21395
WebAuthnErrorTest ++ Only local connections are allowed.
WebAuthnErrorTest ++ Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
WebAuthnErrorTest ++ ChromeDriver was started successfully.
WebAuthnErrorTest ++ 08:32:36,197 INFO  [org.keycloak.testsuite.drone.KeycloakDronePostSetup] Detected browser: chrome-headless-shell 125.0.6422.112
WebAuthnErrorTest ++ 08:32:36,197 INFO  [org.keycloak.testsuite.drone.KeycloakDronePostSetup] Configuring driver settings. implicitWait=2000, pageLoadTimeout=10000
WebAuthnErrorTest ++ 08:32:36,295 DEBUG [org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest] loading test realms
WebAuthnErrorTest ++ 08:32:36,295 INFO  [org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest] importing test realms
WebAuthnErrorTest ++ 08:32:36,295 DEBUG [org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest] --importing realm: test
WebAuthnErrorTest ++ 08:32:37,091 INFO  [org.keycloak.testsuite.util.URLUtils] starting navigation
WebAuthnErrorTest ++ 08:32:37,096 INFO  [org.keycloak.testsuite.util.URLUtils] current URL:  data:,
WebAuthnErrorTest ++ 08:32:37,096 INFO  [org.keycloak.testsuite.util.URLUtils] navigating to https://localhost:8543/auth/realms/master/testing/blank
WebAuthnErrorTest ++ 08:32:37,664 INFO  [org.keycloak.testsuite.util.URLUtils] new current URL:  https://localhost:8543/auth/realms/master/testing/blank
WebAuthnErrorTest ++ 08:32:37,664 INFO  [org.keycloak.testsuite.util.URLUtils] navigation complete
WebAuthnErrorTest ++ 08:32:37,664 INFO  [org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest] deleting cookies in 'master' realm
WebAuthnErrorTest ++ 08:32:37,697 DEBUG [org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest] creating test user
WebAuthnErrorTest ++ 08:32:37,771 INFO  [org.keycloak.testsuite.util.URLUtils] starting navigation
WebAuthnErrorTest ++ 08:32:37,774 INFO  [org.keycloak.testsuite.util.URLUtils] current URL:  https://localhost:8543/auth/realms/master/testing/blank
WebAuthnErrorTest ++ 08:32:37,774 INFO  [org.keycloak.testsuite.util.URLUtils] navigating to https://localhost:8543/auth/realms/test/account/account-security/signing-in
WebAuthnErrorTest ++ 08:32:38,892 INFO  [org.keycloak.testsuite.util.URLUtils] new current URL:  https://localhost:8543/auth/realms/test/protocol/openid-connect/auth?client_id=account-console&redirect_uri=https%3A%2F%2Flocalhost%3A8543%2Fauth%2Frealms%2Ftest%2Faccount%2Faccount-security%2Fsigning-in&state=b5701f38-f056-47b0-b973-9bfce19d53cc&response_mode=query&response_type=code&scope=openid&nonce=20fc424b-b64b-4870-814b-3dc64cd982f4&code_challenge=L_39chD7lLrUJ7fAeJplD4K7IXymrGuKktwpi6QQATY&code_challenge_method=S256
WebAuthnErrorTest ++ 08:32:38,892 INFO  [org.keycloak.testsuite.util.URLUtils] navigation complete
WebAuthnErrorTest ++ 08:32:45,269 INFO  [org.keycloak.testsuite.util.URLUtils] starting navigation
WebAuthnErrorTest ++ 08:32:45,272 INFO  [org.keycloak.testsuite.util.URLUtils] current URL:  https://localhost:8543/auth/realms/test/account/account-security/signing-in
WebAuthnErrorTest ++ 08:32:45,272 INFO  [org.keycloak.testsuite.util.URLUtils] navigating to https://localhost:8543/auth/realms/test/account/account-security/signing-in
WebAuthnErrorTest ++ 08:32:45,275 INFO  [org.keycloak.testsuite.util.URLUtils] target URL equals current URL - refreshing the page
WebAuthnErrorTest ++ 08:32:45,819 INFO  [org.keycloak.testsuite.util.URLUtils] new current URL:  https://localhost:8543/auth/realms/test/account/account-security/signing-in
WebAuthnErrorTest ++ 08:32:45,819 INFO  [org.keycloak.testsuite.util.URLUtils] navigation complete
WebAuthnErrorTest ++ 08:32:54,994 INFO  [org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest] calling all TestCleanup
WebAuthnErrorTest ++ 08:32:55,150 ERROR [org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest] [WebAuthnErrorTest] errorPageWithTimeout() FAILED
WebAuthnErrorTest ++ 08:32:55,151 INFO  [org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest] [WebAuthnErrorTest] errorPageWithTimeout() FINISHED
WebAuthnErrorTest ++ 
WebAuthnErrorTest ++ 
WebAuthnErrorTest ++ 08:32:55,151 INFO  [org.keycloak.testsuite.arquillian.AuthServerTestEnricher] removing test realms after test class
WebAuthnErrorTest ++ 08:32:55,286 INFO  [org.keycloak.testsuite.arquillian.AuthServerTestEnricher] removed realms: test, 

--------- org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest output end  ---------
Error:  Tests run: 3, Failures: 0, Errors: 3, Skipped: 0, Time elapsed: 58.696 s <<< FAILURE! - in org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest
Error: OR] org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest.errorPageWithTimeout  Time elapsed: 20.143 s  <<< ERROR!
-- org.openqa.selenium.NoSuchElementException: 
-- no such element: Unable to locate element: {"method":"xpath","selector":"//*[@data-testid='options']"}
--   (Session info: chrome-headless-shell=125.0.6422.112)
-- For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
-- Build info: version: '4.21.0', revision: '79ed462ef4'
-- System info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.5.0-1021-azure', java.version: '17.0.11'
-- Driver info: org.openqa.selenium.chrome.ChromeDriver
-- Command: [ae5a2c18bcac1d0ba387f1d1de26ca9d, findChildElement {using=xpath, id=f.6DFEF4E176DEFD60C09AF708B40E[729](https://github.com/keycloak/keycloak/actions/runs/9315185575/job/25641016042?pr=30020#step:4:730)D.d.2E4B0A6A17905BDA7899311371A3EAF0.e.43, value=//*[@data-testid='options']}]
-- Capabilities {acceptInsecureCerts: false, browserName: chrome-headless-shell, browserVersion: 125.0.6422.112, chrome: {chromedriverVersion: 125.0.6422.78 (14db42ec38ad..., userDataDir: /tmp/.org.chromium.Chromium...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:33129}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:cdp: ws://localhost:33129/devtoo..., se:cdpVersion: 125.0.6422.112, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
-- Session ID: ae5a2c18bcac1d0ba387f1d1de26ca9d
-- 	at org.jboss.arquillian.graphene.enricher.WebElementUtils$5.getTarget(WebElementUtils.java:127)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneProxyHandler.getTarget(GrapheneProxyHandler.java:161)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler$1.getTarget(GrapheneContextualHandler.java:229)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler$1.invoke(GrapheneContextualHandler.java:180)
-- 	at org.jboss.arquillian.graphene.enricher.SearchContextInterceptor.intercept(SearchContextInterceptor.java:50)
-- 	at org.jboss.arquillian.graphene.proxy.InvocationContextImpl.invoke(InvocationContextImpl.java:87)
-- 	at org.jboss.arquillian.graphene.intercept.InterceptorBuilder$1.intercept(InterceptorBuilder.java:154)
-- 	at org.jboss.arquillian.graphene.proxy.InvocationContextImpl.invoke(InvocationContextImpl.java:87)
-- 	at org.jboss.arquillian.graphene.enricher.StaleElementInterceptor$1.apply(StaleElementInterceptor.java:48)
-- 	at org.jboss.arquillian.graphene.enricher.StaleElementInterceptor$1.apply(StaleElementInterceptor.java:44)
-- 	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:203)
-- 	at org.jboss.arquillian.graphene.wait.WebDriverWaitImpl.until(WebDriverWaitImpl.java:109)
-- 	at org.jboss.arquillian.graphene.enricher.StaleElementInterceptor.intercept(StaleElementInterceptor.java:44)
-- 	at org.jboss.arquillian.graphene.proxy.InvocationContextImpl.invoke(InvocationContextImpl.java:87)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler$2.call(GrapheneContextualHandler.java:252)
-- 	at org.jboss.arquillian.graphene.context.BrowserActions.performAction(BrowserActions.java:62)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler.invoke(GrapheneContextualHandler.java:248)
-- 	at jdk.proxy2/jdk.proxy2.$Proxy67.toString(Unknown Source)
-- 	at java.base/java.lang.String.valueOf(String.java:4220)
-- 	at org.openqa.selenium.support.ui.ExpectedConditions$23.toString(ExpectedConditions.java:669)
-- 	at org.jboss.arquillian.graphene.condition.BooleanConditionWrapper.toString(BooleanConditionWrapper.java:90)
-- 	at java.base/java.lang.String.valueOf(String.java:4220)
-- 	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:225)
-- 	at org.jboss.arquillian.graphene.wait.WebDriverWaitImpl.until(WebDriverWaitImpl.java:109)
-- 	at org.jboss.arquillian.graphene.wait.WebDriverWaitImpl.commit(WebDriverWaitImpl.java:115)
-- 	at org.jboss.arquillian.graphene.wait.IsNotElementBuilderImpl.clickable(IsNotElementBuilderImpl.java:62)
-- 	at org.keycloak.testsuite.util.UIUtils.clickLink(UIUtils.java:76)
-- 	at org.keycloak.testsuite.webauthn.pages.fragments.LoggedInPageHeader.clickOptions(LoggedInPageHeader.java:50)
-- 	at org.keycloak.testsuite.webauthn.pages.fragments.AbstractHeader.clickToolsBtn(AbstractHeader.java:65)
-- 	at org.keycloak.testsuite.webauthn.pages.fragments.AbstractHeader.clickLogoutBtn(AbstractHeader.java:39)
-- 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
-- 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneProxyHandler.invokeReal(GrapheneProxyHandler.java:143)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler$1.invoke(GrapheneContextualHandler.java:180)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler$2.call(GrapheneContextualHandler.java:252)
-- 	at org.jboss.arquillian.graphene.context.BrowserActions.performAction(BrowserActions.java:62)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler.invoke(GrapheneContextualHandler.java:248)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneProxyHandler.intercept(GrapheneProxyHandler.java:106)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneProxyHandler.intercept(GrapheneProxyHandler.java:81)
-- 	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler.intercept(GrapheneContextualHandler.java:92)
-- 	at org.keycloak.testsuite.webauthn.pages.fragments.LoggedInPageHeader_ByGraphene.clickLogoutBtn(Unknown Source)
-- 	at org.keycloak.testsuite.webauthn.account.AbstractWebAuthnAccountTest.logout(AbstractWebAuthnAccountTest.java:169)
-- 	at org.keycloak.testsuite.webauthn.account.WebAuthnErrorTest.errorPageWithTimeout(WebAuthnErrorTest.java:68)
-- 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
-- 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
-- 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
-- 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
-- 	at org.jboss.arquillian.junit.Arquillian$8$1.invokeMethod(Arquillian.java:319)
-- 	at org.jboss.arquillian.junit.MethodInvoker$1.invoke(MethodInvoker.java:18)
-- 	at org.jboss.arquillian.container.test.impl.execution.LocalTestExecuter.execute(LocalTestExecuter.java:57)
-- 	at org.keycloak.testsuite.arquillian.ModelTestExecutor.execute(ModelTestExecutor.java:49)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor613.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.invokeObservers(EventContextImpl.java:103)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:90)
-- 	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:133)
-- 	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:105)
-- 	at org.jboss.arquillian.core.impl.EventImpl.fire(EventImpl.java:62)
-- 	at org.jboss.arquillian.container.test.impl.execution.ClientTestExecuter.execute(ClientTestExecuter.java:50)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor612.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.invokeObservers(EventContextImpl.java:103)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:90)
-- 	at org.jboss.arquillian.container.test.impl.client.ContainerEventController.createContext(ContainerEventController.java:128)
-- 	at org.jboss.arquillian.container.test.impl.client.ContainerEventController.createTestContext(ContainerEventController.java:118)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor611.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.test.impl.TestContextHandler.createTestContext(TestContextHandler.java:116)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor184.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.test.impl.TestContextHandler.createClassContext(TestContextHandler.java:83)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor150.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.test.impl.TestContextHandler.createSuiteContext(TestContextHandler.java:69)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor149.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:133)
-- 	at org.jboss.arquillian.test.impl.EventTestRunnerAdaptor.test(EventTestRunnerAdaptor.java:139)
-- 	at org.jboss.arquillian.junit.MethodInvoker.invoke(MethodInvoker.java:15)
-- 	at org.jboss.arquillian.junit.Arquillian$8.evaluate(Arquillian.java:326)
-- 	at org.jboss.arquillian.junit.Arquillian$4.evaluate(Arquillian.java:198)
-- 	at org.jboss.arquillian.junit.Arquillian.multiExecute(Arquillian.java:344)
-- 	at org.jboss.arquillian.junit.Arquillian.access$200(Arquillian.java:48)
-- 	at org.jboss.arquillian.junit.Arquillian$5.evaluate(Arquillian.java:209)
-- 	at org.jboss.arquillian.junit.Arquillian$7$1.invoke(Arquillian.java:273)
-- 	at org.jboss.arquillian.container.test.impl.execution.ClientBeforeAfterLifecycleEventExecuter.execute(ClientBeforeAfterLifecycleEventExecuter.java:88)
-- 	at org.jboss.arquillian.container.test.impl.execution.ClientBeforeAfterLifecycleEventExecuter.on(ClientBeforeAfterLifecycleEventExecuter.java:66)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor233.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.invokeObservers(EventContextImpl.java:103)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:90)
-- 	at org.jboss.arquillian.container.test.impl.client.ContainerEventController.createContext(ContainerEventController.java:128)
-- 	at org.jboss.arquillian.container.test.impl.client.ContainerEventController.createBeforeContext(ContainerEventController.java:114)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor231.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.test.impl.TestContextHandler.createTestContext(TestContextHandler.java:116)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor184.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.test.impl.TestContextHandler.createClassContext(TestContextHandler.java:83)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor150.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.test.impl.TestContextHandler.createSuiteContext(TestContextHandler.java:69)
-- 	at jdk.internal.reflect.GeneratedMethodAccessor149.invoke(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-- 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
-- 	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
-- 	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
-- 	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:133)
-- 	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:105)
-- 	at org.jboss.arquillian.test.impl.EventTestRunnerAdaptor.fireCustomLifecycle(EventTestRunnerAdaptor.java:159)
-- 	at org.jboss.arquillian.junit.Arquillian$7.evaluate(Arquillian.java:267)
-- 	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
-- 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
-- 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
-- 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
-- 	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
-- 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
-- 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
-- 	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
-- 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
-- 	at org.jboss.arquillian.junit.Arquillian$2.evaluate(Arquillian.java:160)
-- 	at org.jboss.arquillian.junit.Arquillian.multiExecute(Arquillian.java:344)
-- 	at org.jboss.arquillian.junit.Arquillian.access$200(Arquillian.java:48)
-- 	at org.jboss.arquillian.junit.Arquillian$3.evaluate(Arquillian.java:171)
-- 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
-- 	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
-- 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
-- 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
-- 	at org.jboss.arquillian.junit.Arquillian.run(Arquillian.java:109)
-- 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:377)
-- 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:284)
-- 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:248)
-- 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:167)
-- 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
-- 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
-- 	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
-- 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
-- Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@data-testid='options']"}
--   (Session info: chrome-headless-shell=125.0.6422.112)
-- For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
-- Build info: version: '4.21.0', revision: '79ed462ef4'
-- System info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.5.0-1021-azure', java.version: '17.0.11'
-- Driver info: org.openqa.selenium.chrome.ChromeDriver
-- Command: [ae5a2c18bcac1d0ba387f1d1de26ca9d, findChildElement {using=xpath, id=f.6DFEF4E176DEFD60C09AF708B40E729D.d.2E4B0A6A17905BDA7899311371A3EAF0.e.43, value=//*[@data-testid='options']}]
-- Capabilities {acceptInsecureCerts: false, browserName: chrome-headless-shell, browserVersion: 125.0.6422.112, chrome: {chromedriverVersion: 125.0.6422.78 (14db42ec38ad..., userDataDir: /tmp/.org.chromium.Chromium...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:33129}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:cdp: ws://localhost:33129/devtoo..., se:cdpVersion: 125.0.6422.112, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
-- Session ID: ae5a2c18bcac1d0ba387f1d1de26ca9d
-- 	at jdk.internal.reflect.GeneratedConstructorAccessor166.newInstance(Unknown Source)
-- 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
-- 	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
-- 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
-- 	at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
-- 	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
-- 	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
-- 	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
-- 	at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
-- 	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
-- 	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
-- 	at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
-- 	at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
-- 	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
-- 	at org.openqa.selenium.remote.RemoteWebElement.findElement(RemoteWebElement.java:211)
-- 	at org.jboss.arquillian.graphene.GrapheneElementImpl.findElement(GrapheneElementImpl.java:195)
-- 	at org.jboss.arquillian.graphene.GrapheneElementImpl.findElement(GrapheneElementImpl.java:61)
-- 	at org.jboss.arquillian.graphene.enricher.WebElementUtils.dropProxyAndFindElement(WebElementUtils.java:160)
-- 	at org.jboss.arquillian.graphene.enricher.WebElementUtils$5.getTarget(WebElementUtils.java:125)
-- 	... 168 more
```

### Version

999

### Regression

- [X] The issue is a regression

### Expected behavior

Able to find the button

### Actual behavior

Cannot find the button

### How to Reproduce?

Execute a test that accesses the account console.

f.e.

`mvn clean install -Dtest=WebAuthnErrorTest#errorPageWithTimeout -Dbrowser=chrome`

### Anything else?

TIP: to visualize the test, remove the `--headless` arg from the `chromeArguments` in testsuite pom.xml and add some wait utils before the line.

--- a/js/apps/account-ui/src/root/Header.tsx
+++ b/js/apps/account-ui/src/root/Header.tsx
@@ -45,6 +45,7 @@ export const Header = () => {
 
   return (
     <KeycloakMasthead
+      data-testid="page-header"
       keycloak={keycloak}
       features={{ hasManageAccount: false }}
       showNavToggle

--- a/js/libs/ui-shared/src/masthead/KeycloakDropdown.tsx
+++ b/js/libs/ui-shared/src/masthead/KeycloakDropdown.tsx
@@ -8,6 +8,7 @@ import { EllipsisVIcon } from "@patternfly/react-icons";
 import { ReactNode, useState } from "react";
 
 type KeycloakDropdownProps = Omit<DropdownProps, "toggle"> & {
+  "data-testid"?: string;
   isKebab?: boolean;
   title?: ReactNode;
   dropDownItems: ReactNode[];
@@ -29,6 +30,7 @@ export const KeycloakDropdown = ({
       }}
       toggle={(ref) => (
         <MenuToggle
+          data-testid={`${rest["data-testid"]}-toggle`}
           ref={ref}
           onClick={() => setOpen(!open)}
           isExpanded={open}

--- a/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/webauthn/pages/AbstractLoggedInPage.java
+++ b/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/webauthn/pages/AbstractLoggedInPage.java
@@ -36,7 +36,7 @@ import static org.keycloak.testsuite.util.UIUtils.getTextFromElement;
 public abstract class AbstractLoggedInPage extends AbstractAccountPage {
     public static final String ACCOUNT_SECURITY_ID = "account-security";
 
-    @FindBy(xpath = "//div[@id='app']//header[@class='pf-v5-c-page__header']")
+    @FindBy(xpath = "//*[@data-testid='page-header']")
     private LoggedInPageHeader header;
 
     @FindBy(id = "page-sidebar")

--- a/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/webauthn/pages/fragments/AbstractHeader.java
+++ b/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/webauthn/pages/fragments/AbstractHeader.java
@@ -43,23 +43,9 @@ public abstract class AbstractHeader extends AbstractFragmentWithMobileLayout {
         assertToolsBtnVisible(expected, getLogoutBtn());
     }
 
-    public void clickReferrerLink() {
-        clickToolsBtn(getReferrerLink());
-    }
-
-    public void assertReferrerLinkVisible(boolean expected) {
-        assertToolsBtnVisible(expected, getReferrerLink());
-    }
-
-    public String getReferrerLinkText() {
-        return getToolsBtnText(getReferrerLink());
-    }
-
     public abstract void clickOptions();
 
     protected abstract WebElement getLogoutBtn ();
-
-    protected abstract WebElement getReferrerLink();
 
     protected void clickToolsBtn(WebElement btn) {
         clickOptions();

--- a/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/webauthn/pages/fragments/LoggedInPageHeader.java
+++ b/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/webauthn/pages/fragments/LoggedInPageHeader.java
@@ -27,40 +27,24 @@ import static org.keycloak.testsuite.util.UIUtils.getTextFromElement;
  * @author Vaclav Muzikar <vmuzikar@redhat.com>
  */
 public class LoggedInPageHeader extends AbstractHeader {
-    @FindBy(xpath = "//*[@data-testid='options']//a[text() = 'Sign out']")
+    @FindBy(xpath = "//*[@data-testid='page-header']//*[text() = 'Sign out']")
     private WebElement logoutBtn;
-    @FindBy(xpath = "//*[@data-testid='options-kebab']//a[text() = 'Sign out']")
-    private WebElement logoutBtnMobile;
 
-    @FindBy(id = "referrerLink")
-    private WebElement referrerLink;
-    @FindBy(id = "referrerMobileLink")
-    private WebElement referrerLinkMobile;
-
-    @FindBy(xpath = "//*[@data-testid='options']")
+    @FindBy(xpath = "//*[@data-testid='options-toggle']")
     private WebElement options;
-    @FindBy(xpath = "//*[@data-testid='options-kebab']")
-    private WebElement optionsMobile;
-
-    @FindBy(id = "loggedInUser")
-    private WebElement toolbarLoggedInUser;
 
     @Override
     public void clickOptions() {
-        clickLink(isMobileLayout() ? optionsMobile : options);
+        clickLink(options);
     }
 
     @Override
     protected WebElement getLogoutBtn() {
-        return isMobileLayout() ? logoutBtnMobile : logoutBtn;
-    }
 
-    @Override
-    protected WebElement getReferrerLink() {
-        return isMobileLayout() ? referrerLinkMobile : referrerLink;
+        return logoutBtn;
     }
 
     public String getToolbarLoggedInUser() {
-        return getTextFromElement(toolbarLoggedInUser);
+        return getTextFromElement(options);
     }
 }

